src/libs/TransactionUtils.js

```diff
@@ -228,6 +228,16 @@ function getMerchant(transaction) {
     return lodashGet(transaction, 'modifiedMerchant', null) || lodashGet(transaction, 'merchant', '');
 }
 
+/**
+ * Return the category from the transaction. The "created" field has no "modified" complement.
+ *
+ * @param {Object} transaction
+ * @return {*}
+ */
+function getCategory(transaction) {
+    return lodashGet(transaction, 'category', null);
```

Comment:
isnt better to return an empty string if we are passing this to the API?
```suggestion
    return lodashGet(transaction, 'category', '');
```